handsontable/types/core/hooks/index.d.ts

```diff
@@ -167,6 +167,7 @@ export interface Events {
   beforeColumnSort?: (currentSortConfig: ColumnSortingConfig[], destinationSortConfigs: ColumnSortingConfig[]) => void | boolean;
   beforeColumnWrap?: (isActionInterrupted: { value: boolean }, newCoords: CellCoords, isColumnFlipped: boolean) => void;
   beforeColumnUnfreeze?: (columnIndex: number, isUnfreezingPerformed: boolean) => void | boolean;
+  beforeCompositionstart?: (event: CompositionEvent) => void;
```

Comment:
its `beforeCompositionstart` not `beforeCompositionStart` ? like below you have `beforeColumnUnfreeze` not `beforeColumnunfreeze`

Reply:
I used `beforeCompositionstart` name to be similar to the native event name [`compositionstart`](https://developer.mozilla.org/en-US/docs/Web/API/Element/compositionstart_event)